docs/src/__examples__/List/DEFAULT.tsx

```diff
@@ -9,9 +9,25 @@ export default {
       <ListItem>Automobiles</ListItem>
     </List>
   ),
-  info: {
-    title: "Default list",
-    description:
-      "Default lists display their items in primary color with small circles for each item.",
-  },
+  exampleKnobs: [
+    {
+      component: "List",
+      knobs: [
+        { name: "label", type: "text", defaultValue: "" },
+        { name: "icon", type: "icon", defaultValue: "CircleSmall" },
```

Comment:
These should be for `ListItem`, not `List`. We should find a way to automatically prevent these mistakes in the future.

```suggestion
        { name: "label", type: "text", defaultValue: "" },
        { name: "icon", type: "icon", defaultValue: "CircleSmall" },
```